package.json

```diff
@@ -14,7 +14,7 @@
   "dependencies": {
     "@babel/compat-data": "^7.19.1",
     "@mdi/font": "^5.9.55",
-    "array.prototype.move": "0.0.4",
+    "array-move": "4.0.0",
```

Comment:
1. This package file is for Storybook.

2. Why not `^4.0.0` ?

Reply:
I'm thinking you don't need this import ...?

Reply:
Need it for the shares component I thought?

Reply:
So the shares component has its own package file where it depends on it.

THIS package file is for Storybook only (I think).

Reply:
Removed